action.yml

```diff
@@ -7,14 +7,14 @@ inputs:
     description: >-
       The name that the package should be given. This is used as the `--name`
       argument to `raco pkg install`.
-    required: true
+    required: false
   directory:
     description: >-
       The package's root directory, relative to the root of the repository.
-      Should never include a leading slash. For example, a repository with
```

Comment:
Why remove the leading slash thing?

Reply:
I'm passing it through `realpath`, which (among other things) removes extra slashes.